deployment_cfn/.specific/overview.adoc

```diff
@@ -1,5 +1,5 @@
-This Quick Start deploys {partner-product-name} on the AWS Cloud. If you are unfamiliar with AWS Quick Starts we recommend that you read the https://aws-ia.github.io/content/qs_info.html[AWS Quick Start General Content Guide].
+This Quick Start deploys {partner-product-name} on the AWS Cloud. If you are unfamiliar with AWS Quick Starts, see the https://aws-ia.github.io/content/qs_info.html[AWS Quick Start General Content Guide^].
 
-// This deployment guide covers the steps necessary to deploy the Quick Start. For more advanced information on the product, troubleshooting, or additional functionality, see the https://{quickstart-github-org}.github.io/{quickstart-project-name}/operational/index.html[Operational guide].
+// This deployment guide covers the steps necessary to deploy this Quick Start. For more advanced information about the product, troubleshooting, or additional functionality, see the https://{quickstart-github-org}.github.io/{quickstart-project-name}/operational/index.html[Operational guide^].
```

Comment:
Docs are coming to me with no content added to this overview doc—only the boilerplate cross-ref to the GC guide (no actual overview). We could help SAs out by giving them a fill-in-the-blank statement, as we do in `architecture.adoc`.